app/frontend/types.ts

```diff
@@ -78,6 +78,24 @@ export type Token = Omit<TokenObject, 'quantity'> & {
   quantity: number
 }
 
+export type RegisteredTokenMetadata = {
+  subject: string
+  description: string
+  ticker?: string
+  url?: string
+  logoHex?: string
+  decimals?: number
+}
+
+export type UnregisteredToken = Token & {
```

Comment:
I think this type neither `RegisteredToken` isn't currently used, right?

Reply:
I removed dead code